source/plugin/advanced/config-in-depth/paths.rst

```diff
@@ -0,0 +1,50 @@
+========================
+Navigating Through Nodes
+========================
+
+.. note::
+
+    The following discussion about paths uses a period (.) as a separator character. This is not always the character used, but it is a common way of denoting the next node in a path.
+
+Let's discuss how node paths work first. They're just like file paths, a collection of nodes, separated by a single character, leading to the final child node that holds a value, whether that value be a normal value, a list, or simply another node. As an example, ``config.blockCheats.enabled`` is a path, leading to the node ``enabled``, under ``blockCheats``, which is under ``config``. In a configuration file, this would appear like so:
+
+.. code-block:: json
+
+    config {
+        blockCheats {
+            enabled=value
+        }
+    }
+
+Let's figure out how to result in this file through code. We're aiming for the node ``config.blockCheats.enabled`` to return a boolean value of true.
+
+Moving Around
+~~~~~~~~~~~~~
+
+.. tip::
+
+    To make node retrieval easier (and to give less load on the system), it is advised to store a reference to the root node of your configuration file. This allows you to always be able to get the node using the full path, and avoids using the ``getPath()`` method, which is said to be a little intensive by official documentation.
+
+Let's start off by defining our root node again. We'll be using a blank node to write on.
+
+.. code-block:: java
+
+    ConfigurationNode rootNode = loader.createEmptyNode(ConfigurationOptions.defaults());
+
+The method to get a node from a node is the ``getNode(Object...)`` method. Note that ``Object...`` type for the parameter, as this means you cannot simply just give it a path like ``config.blockCheats.enabled``. Doing so results in a config file like this:
+
+.. code-block:: json
+
+    config.blockCheats.enabled=value
+
+... which is not the organized intended result we're looking for. Instead, what you -can- do is split the path by the separator character so it ends up as 3 separate arguments.
+
+.. code-block:: java
+
+    rootNode.getNode((Object[]) "config.blockCheats.enabled".split("\\."));
```

Comment:
I did do prior testing and it does work with that method, and there isn't much danger from doing it that way, from the looks of it.